src/backup_domain.rs

```diff
@@ -22,3 +22,57 @@ use crate::pac::BKP;
 pub struct BackupDomain {
     pub(crate) _regs: BKP,
 }
+
+macro_rules! write_drx {
+    ($self:ident, $drx:ident, $idx:expr, $new:expr) => {
+        unsafe {
+            $self._regs.$drx[$idx].write(|w| w.d().bits($new));
+        }
+    };
+}
+
+macro_rules! read_drx {
+    ($self:ident, $drx:ident, $idx:expr) => {
+        $self._regs.$drx[$idx].read().d().bits();
+    };
+}
+
+impl BackupDomain {
+    /// Read a 16-bit value from one of the DR1 to DR10 registers part of the
+    /// Backup Data Register. The register argument is a zero based index to the
+    /// DRx registers: 0 is DR1, up to 9 for DR10. Providing a number above 9
+    /// will panic.
+    pub fn read_data_register_low(&self, register: usize) -> u16 {
+        assert!(register < 10);
```

Comment:
Why you add assert here?
`dr[register]` will panic with `index out of bounds`

Reply:
OK, I'll remove it.